docs/command-reference/server-management/client-tracking.md

```diff
@@ -0,0 +1,70 @@
+---
+description:  Learn how to use Redis CLIENT TRACKING command to control server-assisted client side caching for the connection.
+---
+
+import PageTitle from '@site/src/components/PageTitle';
+
+# CLIENT TRACKING
+
+<PageTitle title="Redis CLIENT TRACKING Command (Documentation) | Dragonfly" />
+
+## Syntax
+
+    CLIENT TRACKING <ON | OFF>
+
+**Time complexity:** O(1). Some options may introduce additional complexity.
+
+**ACL categories:** @slow, @connection
+
+**Important**: New in Dragonfly v1.14, only available when the [RESP3](https://github.com/redis/redis-specifications/blob/master/protocol/RESP3.md) protocol is used.
+
+The `CLIENT TRACKING` command enables the tracking feature of the Dragonfly server, which is used for server-assisted client side caching.
+See more details about server-assisted client side caching in the [Redis documentation](https://redis.io/docs/manual/client-side-caching/).
+
+When tracking is enabled, Dragonfly remembers the keys that the connection requested in order to send later invalidation messages when such keys are modified.
```

Comment:
and we need to mention that only a read command after tracking is enabled can activate the tracking for the key. 

so for example, if you turn on client tracking, and just set the key, this key will not be tracked until it was read using get.